extensions/ctrpull/ignore.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ctrpull
+
+var ignored map[string]struct{}
+
+// SetClusterKind sets ignored for cluster type `kind`
+func SetClusterKind() {
```

Comment:
Why we can not use just a  regexp flag of ignoring applications?

Reply:
1. List approach looks better for the reader.
2. There is actually no profit in using some unpleasant regexp instead of list.

Reply:
1. Do not see problems here. Could you explain?
2. With this approach we could ignore a class of applications like `sriov` or `vfio`. So in case of adding a new application for sriov we'll not need to change the list of ignored applications...

Reply:
Yep, replaced with regexps and flags.